.github/workflows/ci.yml

```diff
@@ -1,14 +1,23 @@
-name: Build
+name: Continuous Integration
 
 on:
   workflow_dispatch:
   push:
-    branches: [ master ]
+    branches:
+      - main
+  pull_request:
```

Comment:
We actually don't want pull requests built I think. At least for the time being. The reason is that people could write bots that maliciously open a lot of PRs. So I would opt for having `pull_requests:` without any nested values, just like in the other repos.

Reply:
I am not an expert in Github Actions, but wouldn't this also run the workflow on pull requests ? And bots can still abuse it ? Anyway I applied your suggestion. This would also match other repos as you mentioned.